app/cypress/integration/cif/project-revision/create-first-project-revision.spec.js

```diff
@@ -178,15 +178,7 @@ describe("when creating a project, the project page", () => {
       "20 %"
     );
     // additional funding sources section
-    cy.findByText(/Additional Funding Source 1/i)
-      .next()
-      .should("have.text", "Test Source 1");
-    cy.findByText(/Additional Funding Amount \(Source 1\)/i)
-      .next()
-      .should("have.text", "$111.00");
-    cy.findByText(/Additional Funding Status \(Source 1\)/i)
-      .next()
-      .should("have.text", "Approved");
+    cy.findByText(/Additional Funding Source 1/i).should("be.visible");
```

Comment:
The tech debt ticket I added covers rewriting these once the numbering is added back in: https://app.zenhub.com/workspaces/climate-action-secretariat-60ca4121764d710011481ca2/issues/gh/bcgov/cas-cif/1535